### PR TITLE
Adding missing security test

### PR DESCRIPTION
**Pull request by ahmetakbn**

### CODE REVIEW

**Comment by OskarStark** [comment]
Wow thank you for the test coverage @ahmetakbn 👍 

@greg0ire @Soullivaneuh would be cool to get travis running, to check if the tests are working.

@greg0ire @Soullivaneuh do we need a changelog note for this too, I'd say no!

File: Security/SessionDownloadStrategy.php [hunk 1]
```diff
@@ -15,6 +15,7 @@
 use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\HttpFoundation\Session\Session;
```
Comment (OskarStark) [inline Security/SessionDownloadStrategy.php]: is this really needed @ahmetakbn ?

Comment (OskarStark) [inline Security/SessionDownloadStrategy.php]: Sorry, its used hereand correct

https://github.com/ahmetakbn/SonataMediaBundle/blob/01bbb0658c66c4ac24699b44a80c7cd9d0101ed3/Security/SessionDownloadStrategy.php#L85

only the order needs to be fixed

**Comment by Soullivaneuh** [comment]
> would be cool to get travis running, to check if the tests are working.

I'm doing the best I can for this.

BTW, it's not a critical PR, this can wait a little.

> do we need a changelog note for this too, I'd say no!

No nobody don't give a f*ck about test update. :trollface:

File: Tests/Security/SessionDownloadStrategyTest.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Security;
+
+use Sonata\MediaBundle\Security\SessionDownloadStrategy;
+
+class SessionDownloadStrategyTest extends \PHPUnit_Framework_TestCase
```
Comment (Soullivaneuh) [inline Tests/Security/SessionDownloadStrategyTest.php]: * `final`
* Add a class comment with author tag please.

**Comment by OskarStark** [comment]
> No nobody don't give a f*ck about test update. :trollface:

😄 

> I'm doing the best I can for this.

i know 👍 

> BTW, it's not a critical PR, this can wait a little.

Yes!

**Comment by OskarStark** [comment]
@ahmetakbn there is another CS error. Please check `SonataCI - Quick Check` details!

**Comment by ahmetakbn** [comment]
> @ahmetakbn there is another CS error. Please check SonataCI - Quick Check details!

Yes, I saw. Just was able to check. Fixed now.

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.

**Comment by OskarStark** [comment]
@ahmetakbn please create this PR against `3.x` thank you

**Pull request closed without merging**